Dockerfile.agent

```diff
@@ -1,23 +1,23 @@
-FROM python:3.7 AS builder
-
-RUN pip install pybuilder --pre
-
-COPY api /var/xdcs/api
-COPY xdcs-agent /var/xdcs/xdcs-agent
-
-WORKDIR /var/xdcs/xdcs-agent
-
-RUN pyb install_build_dependencies && \
-    pyb publish
-
-################################################################################
-
-FROM python:3.7
-
-COPY --from=builder /var/xdcs/xdcs-agent/target/dist/xdcs-agent-*/dist/xdcs-agent-*.tar.gz \
-    /var/xdcs-agent/xdcs-agent.tar.gz
-
-WORKDIR /var/xdcs-agent
-RUN pip install xdcs-agent.tar.gz && rm xdcs-agent.tar.gz
-
-CMD xdcs-agent
+FROM python:3.7 AS builder
+
+RUN pip install pybuilder --pre
+
+COPY api /var/xdcs/api
+COPY xdcs-agent /var/xdcs/xdcs-agent
+
+WORKDIR /var/xdcs/xdcs-agent
+
+RUN pyb -X install_build_dependencies && pyb -X install_dependencies && \

```

Comment:
do nowej linii

Reply:
a tak w ogóle to chyba install_dependencies nie powinno być potrzebne bo w tym dockerze nie uruchamiamy agenta tylko go budujemy